### PR TITLE
Package ppx_pbt.0.2.0

### PR DESCRIPTION
**Pull request by vch9**
### `ppx_pbt.0.2.0`
PPX Rewriter for property based testing

---
* Homepage: https://gitlab.com/vch9/ppx_pbt
* Source repo: git+https://gitlab.com/vch9/ppx_pbt.git
* Bug tracker: https://gitlab.com/vch9/ppx_pbt/-/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ppx_pbt/ppx_pbt.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "data-encoding" {>= "0.4"}
+  "zarith" {>= "1.12"}
+  "qcheck-alcotest" {>= "0.17"}
+  "ppx_deriving_qcheck" {>= "0.1"}
+  "ppx_deriving" {>= "5.0"}
+]
+
+pin-depends: [
+  ["ppx_deriving_qcheck.dev" "git+https://github.com/vch9/ppx_deriving_qcheck.git#dev"]
+]
```
Comment (kit-ty-kate) [inline packages/ppx_pbt/ppx_pbt.0.2.0/opam:31]: pin-depends are not allowed in opam-repository. Could you release it first?

Comment (vch9) [reply packages/ppx_pbt/ppx_pbt.0.2.0/opam:31]: Oh well, I'm closing this until I release it. Thank you

**Comment by kit-ty-kate** [comment]
The checksum given doesn't seem to match the archive:
```

[ERROR] The sources of the following couldn't be obtained, aborting:
          - ppx_pbt.0.2.0:
            https://gitlab.com/vch9/ppx_pbt/-/archive/v0.2.0/ppx_pbt-v0.2.1.tar.gz (Bad checksum, expected md5=2bf1733eaca646a1fd0eac136e934df0)
```

**Comment by kit-ty-kate** [comment]
the version number is taken from the directory name of the file added, so you just need to move the directory to the correct version

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fix in the opam file upstream so that it can be taken into account for next time?

**Pull request merged**